docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -938,7 +938,7 @@ for example, when the string is `NULL` or it does not contain valid JSON,
 or the JSON value is not an object.
 
 ```sql title="Examples"
-JSON_KEYS("{\"a\": \"abc\", \"b\": { \"c\": \"a\" }, \"d\": 1}") => ["a", "b", "d"]
+JSON_KEYS('{\"a\": \"abc\", \"b\": { \"c\": \"a\" }, \"d\": 1}') => [a, b, d]
```

Comment:
Should this output be `["a", "b", "d"]`, even though the ksqlDb CLI outputs it as `[a, b, d]`?